pom.xml

```diff
@@ -113,9 +113,15 @@
             <version>1.7R5</version>
         </dependency>
         <dependency>
-          <groupId>com.jayway.jsonpath</groupId>
-          <artifactId>json-path</artifactId>
-          <version>2.0.0</version>
+            <groupId>com.jayway.jsonpath</groupId>
+            <artifactId>json-path</artifactId>
+            <version>2.0.0</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>net.minidev</groupId>
+                    <artifactId>asm</artifactId>
```

Comment:
I don't think so, since it's linked against by a static "DEFAULTS" in one of the jsonpath configuration classes.